pkg/cache/cache.go

```diff
@@ -162,31 +173,27 @@ func isPackageContent(content configapi.RepositoryContent) bool {
 	return content == configapi.RepositoryContentPackage
 }
 
-func (c *Cache) CloseRepository(repositorySpec *configapi.Repository) error {
-	var key string
+func (c *Cache) CloseRepository(repositorySpec *configapi.Repository, allRepos []configapi.Repository) error {
```

Comment:
In theory I absolutely agree that we should have both. I will add at least some unit tests

Reply:
I've modified cache unit tests by adding coverage for CloseRepository().
I haven't add any tests that check the current cache-key generation behavior, since it is expected to change soon.